src/SQLQueryStress/SQLQueryStress.csproj

```diff
@@ -1,31 +1,18 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="12.0">
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>8.0.50727</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{02A166A4-EE64-432F-AA6E-08C7984BD7BC}</ProjectGuid>
+    <TargetFramework>netcoreapp3.1</TargetFramework>

```

Comment:
Make it an .exe

Reply:
It must be an .exe, is there a problem with the build?    
<TargetFramework>netcoreapp3.1</TargetFramework>
<OutputType>WinExe</OutputType>

Reply:
I think I just go confused by looking at the build log.